proposals/lip-0045.md

```diff
@@ -224,6 +224,7 @@ In this section, we specify the substores that are part of the Interoperability
 | `EMPTY_FEE_ADDRESS` | bytes | "" | The empty byte string. |
 | `MIN_RETURN_FEE_PER_BYTE_BEDDOWS` | uint64 | 1000 | The minimum return fee per byte for a cross-chain message in Beddows for channels between the mainchain and a sidechain. |
 | `MAX_CCM_SIZE` | uint32 | 10*1024 | The maximum size of a serialized cross-chain message in bytes. |
+| `RECOVERED_STORE_VALUE` | bytes | `\x00 * 32` | The constant value used for recovered entries of the state recovery command. Since there is no known sha256 pre-image for this constant, the same store entry cannot be recovered twice. |
```

Comment:
You are not using an apostrophe here. Further, I would suggest using the same notation as in other LIPs, e.g., LIP 63.